pkg/cmd/render/render.go

```diff
@@ -192,7 +206,7 @@ func (r *renderOpts) Run() error {
 		BindAddress:                   "0.0.0.0:6443",
 		BindNetwork:                   "tcp4",
 		TerminationGracePeriodSeconds: 135, // bit more than 70s (minimal termination period) + 60s (apiserver graceful termination)
-		ShutdownDelayDuration:         "",  // do not override
+		ShutdownDelayDuration:         r.shutdownDelayDuration.String(),
```

Comment:
BTW: Do we want to expose it as a command line arg ? If yes then we have to coordinate updating it with `terminationGracePeriodSeconds`. Otherwise `kubelet` might terminate `kas` process before graceful termination completes.